packages/terra-application-docs/src/terra-dev-site/tool/terra-dev-site/UpgradeGuides.f/v2.0.0.q.tool.mdx

```diff
@@ -0,0 +1,10 @@
+# v2.0.0 Upgrade Guide
+This document will provide information on upgrading from Terra Application `^1.0.0` to `2.0.0`.
+
+## Dependency updates
+
+Terra Application was updated to use React-Intl `v5`. Also, Dropped support to older versions of React-Intl.
+Projects consuming terra-application version 2 should use React Intl version >= 5.
+
+React-intl V5 no longer exports the `intlShape` consuming project should replace the `intlShape` with `PropTypes.shape({ formatMessage: PropTypes.func })`. See [doc](https://formatjs.io/docs/intl#the-intl-object) for most complete and up to date intl shape.
+Follow the upgrade guide of react-intl v5 to know more about the breaking changes : https://formatjs.io/docs/react-intl/upgrade-guide-5x 
```

Comment:
Most of the breaking changes happened in v3, I would link to the upgrade guides for v3 and v4 in addition to the v5 link so consumers can find all the breaking changes they need to be aware of for this jump.